vnext/Microsoft.ReactNative/Fabric/platform/react/renderer/graphics/Color.cpp

```diff
@@ -7,58 +7,113 @@
 namespace facebook::react {
 
 bool isColorMeaningful(SharedColor const &color) noexcept {
-  if (!color) {
-    return false;
+    if (!color) {
+      return false;
+    }
+
+    return colorComponentsFromColor(color).alpha > 0;
+  }
+
+winrt::Windows::UI::Color ResolvePlatformColor(Color const * const color) {
+    // Issue #11489. These are all the light-theme values. Which is better than no values.
```

Comment:
Can you add a comment on *where* you got these values from, so we know where to look in the future?

Reply:
Added comment with description and file url for both shared colors and control colors.